src/Umbraco.Web.UI.Client/src/views/propertyeditors/datepicker/datepicker.controller.js

```diff
@@ -78,9 +78,20 @@ function dateTimePickerController($scope, notificationsService, assetsService, a
         setDatePickerVal();
     };
 
-    $scope.inputChanged = function() {
-        setDate($scope.model.datetimePickerValue);
-        setDatePickerVal();
+    $scope.inputChanged = function () {        
+        if ($scope.model.datetimePickerValue == "" && $scope.hasDatetimePickerValue) {
+                $scope.clearDate();
+        } else if ($scope.model.datetimePickerValue) {
+            var momentDate = moment($scope.model.datetimePickerValue, $scope.model.config.format, true);
+            if (!momentDate || !momentDate.isValid()) {
+                momentDate = moment(new Date($scope.model.datetimePickerValue));
+            }
+            if (momentDate && momentDate.isValid()) {
+                setDate(momentDate.format("YYYY-MM-DD HH:mm:ss"));
```

Comment:
Why do we hardcode a format here instead of using the one from `$scope.model.config.format`?